Makefile

```diff
@@ -8,7 +8,7 @@ format:
 	@poetry run black .
 
 lint:
-	@poetry run pylint ./seam ./test
+	@poetry run pylint ./seam ./test --rcfile ./.pylintrc
```

Comment:
I thought it reads this file by default. Can you try removing this and renaming from `.pylintrc` to `pylintrc`?

Reply:
I worked without renaming. I wonder why it didn't work the first time I tried

Reply:
https://github.com/seamapi/python-next/pull/6/commits/3d439f33d5614e4ad441a76c386f75848fe72648

Reply:
Weird. The official name seems to be `pylintrc` without the `.` Can we rename it to that and make sure it still works? https://github.com/pylint-dev/pylint/blob/main/pylintrc

Reply:
https://github.com/seamapi/python-next/pull/6/commits/77cc58561b238b4c8497ce79034333a510a48650